src/Node/LavaNode.cs

```diff
@@ -400,25 +400,25 @@ await voiceChannel.DisconnectAsync()
                             return;
                         }
 
+                        var playerTrack = player.Track;
```

Comment:
why is there another playerTrack and why isn't lavaTrack being used for events?

Reply:
> 

`lavaTrack` does not have the type TLavaTrack (One of the reasons Update(LavaTrack) functions was added to keep it happy), so cannot box this to events, hence we unbox `player.Track` to `playerTrack` and box this to the events that needs it